cpp/ql/src/experimental/query-tests/Security/CWE/CWE-409/DecompressionBombs.ql

```diff
@@ -0,0 +1,43 @@
+/**
+ * @name User-controlled file decompression
+ * @description User-controlled data that flows into decompression library APIs without checking the compression rate is dangerous
+ * @kind path-problem
+ * @problem.severity error
+ * @security-severity 7.8
+ * @precision high
+ * @id cpp/data-decompression
+ * @tags security
+ *       experimental
+ *       external/cwe/cwe-409
+ */
+
+import cpp
+import semmle.code.cpp.ir.dataflow.TaintTracking
+import semmle.code.cpp.security.FlowSources
+import semmle.code.cpp.commons.File
+import MiniZip
+import ZlibGzopen
+import DecompressionBomb
```

Comment:
This now seems to miss some of the needed imports to ensure all the extensions of ` DecompressionBomb` are in scope.